scss/_utilities.scss

```diff
@@ -100,7 +100,15 @@ $utilities: map-merge(
     "border-color": (
       property: border-color,
       class: border,
-      values: map-merge($theme-colors, ("white": $white))
+      values: map-merge(
+        $theme-colors,
+        (
```

Comment:
What about using a dedicated variable here, something like `$background-colors` to handle the map and ease customisation?